.hlint.yaml

```diff
@@ -5,8 +5,6 @@
 - ignore: {name: "Redundant bang pattern"} # 33 hints
 - ignore: {name: "Redundant flip"} # 1 hint
 - ignore: {name: "Redundant if"} # 1 hint
-- ignore: {name: "Use $>"} # 7 hints
-- ignore: {name: "Use <$"} # 1 hint
```

Comment:
I find expressions `$>` and `<$` harder to read. Might be just me not familiarized with this. But I don't think I'll oppose merging if someone wants argue otherwise.